microcosm_pubsub/chain/context_decorators.py

```diff
@@ -0,0 +1,101 @@
+from functools import wraps
+from marshmallow import ValidationError
+from inspect import getfullargspec, ismethod, isfunction
+
+from microcosm_pubsub.chain.decorators import EXTRACTS, BINDS
+
+
+def get_from_context(context, func):
+    """
+    Decorate a function - pass to the function the relevant arguments
+    from a context (dictionary) - based on the function arg names
+
+    """
+    include_self = int(ismethod(func) or not isfunction(func))
```

Comment:
`or not isfunction`: for callable class instances

Reply:
s/int/bool/